assets/source/setup-guide/app/helpers/documentation-link-props.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * External dependencies
+ */
+import { recordEvent } from '@woocommerce/tracks';
+
+/**
+ * Clicking on an external documentation link.
+ *
+ * @event wcadmin_pfw_documentation_link_click
+ *
+ * @property {string} link_id Identifier of the link.
+ * @property {string} context What action was initiated.
+ * @property {string} href Href to which the user was navigated to.
+ */
+
+/**
+ * Creates properties for an external documentation link.
+ * May take any other props to be extended and forwarded to a link element (`<a>`, `<Button isLink>`).
+ *
+ * Sets `target="_blank" rel="noreferrer"` and onClick handler that fires track event.
+ *
+ * @fires pfw_documentation_link_click on click, with given `linkId` and `context`.
+ *
+ * @param {Object} props React props.
+ * @param {string} props.href Href to used by link and in track event.
+ * @param {string} props.linkId Forwarded to {@link wcadmin_pfw_documentation_link_click}
+ * @param {string} props.context Forwarded to {@link wcadmin_pfw_documentation_link_click}
+ * @param {string} [props.target='_blank']
+ * @param {string} [props.rel='noreferrer']
+ * @param {Function} props.onClick
+ * @param {...import('react').AnchorHTMLAttributes} props.props
+ */
+export default function documentationLinkProps( {
+	href,
+	linkId,
+	context,
+	target = '_blank',
+	rel = 'noreferrer',
+	onClick,
+	...props
+} ) {
+	return {
+		href,
+		target,
+		rel,
```

Comment:
📜 
I was thinking would it be better if use `'noopener'` as the base `rel` value internally when the `target` is `'_blank'`, and then the passing `rel` is combined and deduplicated as the final `rel`? Thus, `documentationLinkProps( { ref: 'noreferrer' } )` would return `{ ref: 'noopener noreferrer' }`.

Reply:
Like a polyfill for IE? I like the idea!

Reply:
After a second thought, I think that maybe it's too much magic. Also, it would introduce another concern similar to https://github.com/woocommerce/pinterest-for-woocommerce/pull/266#discussion_r751915176 and more:

What about 
1. `<MyElement target="_blank" {...documentationLinkProps}/>`
2. `<MyElement {...documentationLinkProps( { target="_blank", rel: false} )} />`
3. `<MyElement {...documentationLinkProps( { target="_blank", rel: ''} )} />`

WDYT of making it more straightforward, and a bit opposite to what we have now?:

`documentationLinkProps` will set `rel: noopener` by default. Then the number of current use-cases, would set it to `noreferrer` ([which implies `noopener`](https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/noreferrer))

BTW, I'm not sure if those use cases should use `noreferrer`, as I thought Pinterest could be interested to track referrers to their docs.

Reply:
> WDYT of making it more straightforward, and a bit opposite to what we have now?

I think that's good to make it straightforward and take `noopener` as default value. 👍 

> BTW, I'm not sure if those use cases should use `noreferrer`, as I thought Pinterest could be interested to track referrers to their docs.

I don't know either. Staying the same as the current codebase should be ok. And if Pinterest side needs this we can do it by separate PR.